src/app.ts

```diff
@@ -140,12 +145,15 @@ async function runBuild(input: string, program: OptionValues, helpText: string)
     process.exit(1);
   }
 
-  if (program.debug) logger.level = 'debug';
+  // Set the log level. If no level is specified, logger defaults to info
+  if (program.logLevel === 'debug' || program.logLevel === 'warn' || program.logLevel === 'error') {
```

Comment:
Now that commander halts on invalid log-levels, we don't need to check them all here. Instead we only need to change the level if it's _not_ the default.
```suggestion
  if (program.logLevel !== 'info') {
```

Reply:
Ah good call!

Reply:
[951fac3](https://github.com/FHIR/sushi/pull/1208/commits/951fac3ba456e505e7fbbc91294cb8a930bdac3c) updates the `if`. It also simplifies the description of the option to work better with the choices.